### ISSUE TITLE
Endpoint doesn't work for Guest users. However another undocumented endpoint works.

### ISSUE THREAD

**Issue by vman** [microsoftgraph/microsoft-graph-docs#9182]
The docs say the endpoint is `v1.0/me/joinedTeams`

This works for regular users, but not for guest users.
 
For guest users (as well as regular users), this endpoint seems to work `v1.0/users/me/joinedTeams`
 
But this endpoint is not documented anywhere. I just found it randomly while playing around with postman.

Should the `v1.0/me/joinedTeams` endpoint work for guest users or should we stick to using the undocumented endpoint?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1fbcfbc4-78ba-8b4c-d5b9-dc58f9d84f5f
* Version Independent ID: 1dcb5dda-2a45-77e1-bbe1-46ec24b33eec
* Content: [List joinedTeams - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/user-list-joinedteams?view=graph-rest-1.0&tabs=http)
* Content Source: [api-reference/v1.0/api/user-list-joinedteams.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/user-list-joinedteams.md)
* Product: **microsoft-teams**
* Technology: **microsoft-graph**
* GitHub Login: @nkramer
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by Nikitha-MSFT** [comment]
v1.0/users/me/joinedTeams is not a valid API. Please don't use it, we have no plans to support it and it is breaks we will not fix it. As far as the Guest user support is concerned, we are looking into documenting internally the set of APIs which supports Guest users.

**Comment by vman** [comment]
Thanks for your response @Nikitha-MSFT. Looking forward to the documentation of the Guest user supported API endpoints.

**Comment by vman** [comment]
Hi @Nikitha-MSFT , is there any update on this issue to see which endpoints are supported for guest users? Thanks!

**Comment by Nikitha-MSFT** [comment]
We are looking into it. We don't have ETA to share when it will available.

**Comment by rohannevrikar** [comment]
Hi! I have been trying to add guest users to a team using `/teams/{teamId}/members` endpoint. However I'm getting 403 Forbidden error. When I tried the same endpoint for a "member" type user, it worked as expected. I have used application permission TeamMember.ReadWrite.All. @vman by any chance did you come across this or face similar issue? Or am I missing something here? Or is this endpoint too not supported by guest users? I have been following this doc: [Add members to team](https://docs.microsoft.com/en-us/graph/api/team-post-members?view=graph-rest-1.0&tabs=http)
